subsys/bluetooth/shell/audio.c

```diff
@@ -1409,6 +1409,34 @@ static int cmd_term_broadcast_sink(const struct shell *sh, size_t argc,
 }
 #endif /* CONFIG_BT_AUDIO_BROADCAST_SINK */
 
+static int cmd_context(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err;
+	uint8_t ctxt;
+	int32_t dir;
+
+	ctx_shell = sh;
+
+	if (!strcmp(argv[1], "sink")) {
+		dir = BT_AUDIO_DIR_SINK;
+	} else if (!strcmp(argv[1], "source")) {
+		dir = BT_AUDIO_DIR_SOURCE;
+	} else {
+		shell_error(sh, "Unsupported dir: %s", argv[1]);
+		return -ENOEXEC;
+	}
+
+	ctxt = strtol(argv[2], NULL, 0);
```

Comment:
Please use the `shell_strtol` when getting shell commands 

This should probably also be the unsigned version.

Reply:
Done